README.md

```diff
@@ -57,7 +57,7 @@ BenchmarkComparisonXXHash-8                	200000000	        89 ns/op	  89.28 M
 ## Filling a slice by index or append
 
 ```
-BenchmarkSliceFillByIndex-8       	 2000000	       623 ns/op
-BenchmarkSliceFillByIndexMake-8   	 2000000	       626 ns/op
-BenchmarkSliceFillAppend-8        	 1000000	      1304 ns/op
+BenchmarkSliceFillByIndex-4         	 2000000	       627 ns/op
+BenchmarkSliceFillByIndexMake-4     	 2000000	       631 ns/op
+BenchmarkSliceFillMakeAppend-4      	 2000000	       902 ns/op
```

Comment:
Oh I should have mentioned that I ran those on linux with `go version go1.9beta1 linux/amd64` 

I reran them with `go version go1.8.3 linux/amd64` where I see similar results like you (i.e. `1309 ns/op`)